### DISCUSSION TITLE
Sorting JSON value [transform]

### DISCUSSION THREAD

**Discussion by RFskynet** [vectordotdev/vector#20436] [Q&A] [answered]
Hi. After looking at the documentation, I did not find an answer to my question. How can I sort the values of the groups alphabetically?
Example:
{
   "json": {
       "groups": [
              "group2",
              "group3",
              "group1"
                       ]
              }
}

**Comment by jszwedko** [comment, marked as answer]
Unfortunately there is no way to do that in Vector natively. You could use a `lua` transform which would give you access to Lua which has sort functions.
